runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -78,7 +91,9 @@
 @Slf4j
 public class RaidsPlugin extends Plugin
 {
+	private static final DateFormat TIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
```

Comment:
Instead of duplicating this across plugins which will share the screenshot utility, we could define this as a constant in the screenshot util.